pyanaconda/ui/gui/spokes/advstorage/iscsi.py

```diff
@@ -19,94 +19,27 @@
 
 from collections import namedtuple
 
-from pyanaconda.core import constants
-from pyanaconda.threading import threadMgr, AnacondaThread
-from pyanaconda.core.timer import Timer
+from pyanaconda.modules.common.errors.configuration import StorageDiscoveryError
+from pyanaconda.modules.common.task import async_run_task
+from pyanaconda.modules.common.structures.iscsi import Credentials, Portal, Node
+from pyanaconda.modules.common.constants.services import STORAGE
+from pyanaconda.modules.common.constants.objects import ISCSI
+from pyanaconda.modules.storage.constants import IscsiInterfacesMode
```

Comment:
You should use constants from `pyanaconda.core.constants` here. UI shouldn't import anything from the DBus modules (unless it is a temporary workaround).

Reply:
https://github.com/rhinstaller/anaconda/pull/1960/commits/de96c793ec1d255c991ddf355410865846883d9a